### DISCUSSION TITLE
How would I go about displaying people results from an M365 Group.

### DISCUSSION THREAD

**Discussion by dillon-obrien** [microsoft-search/pnp-modern-search#3528] [Q&A] [answered]
How would I go about displaying people results from an M365 Group.

What API permissions would I need?
Is this Possible or do I have to rely only on User Profile Services?

**Comment by kasperbolarsen** [comment]
What is your goal, showing Members and/or Owners in same results web part?

**Reply by wobba** [reply, marked as answer]
..and you cannot use search API's for this. You need to list owners/members via the Graph groups API.
